.github/workflows/sycl_linux_build_and_test.yml

```diff
@@ -64,38 +64,11 @@ on:
         type: string
         required: false
         default: 'SYCL-2020'
-      mlir_sycl_doc:
```

Comment:
Are we not compiling the docs in CI anymore?

Reply:
We are, see line 120. 
These are input variables that determine if checks should be run.
The idea is moved to sycl_detect_changes, and under the new variable check_filters.